src/renderers/shared/fiber/__tests__/ReactIncrementalScheduling-test.js

```diff
@@ -340,4 +340,13 @@ describe('ReactIncrementalScheduling', () => {
     // animation priority.
     expect(ReactNoop.getChildren()).toEqual([span(1)]);
   });
+
+  it('can force synchronous updates with syncUpdates, even inside batchedUpdates', () => {
+    ReactNoop.batchedUpdates(() => {
+      ReactNoop.syncUpdates(() => {
+        ReactNoop.render(<span />);
+        expect(ReactNoop.getChildren()).toEqual([span()]);
```

Comment:
Oh I see. Yeah this test assumes that `batchedUpdates` works properly, because we have other tests that cover the behavior `batchedUpdates`.